README.md

```diff
@@ -1,4 +1,84 @@
-# Facilities-Locator-post-11-11-2015
-Placeholder repository containing next gen Facilities Locator artifacts and eventual application. 
+# Service And Facility Locator
 
-Created to centralize related tasks and management efforts. 
+This build monitor should be up and running as soon as you set up travis!
+[![Build Status](https://travis-ci.org/department-of-veterans-affairs/service-and-facility-locator.svg?branch=master)](https://travis-ci.org/department-of-veterans-affairs/service-and-facility-locator)
+
+##Setup for your local environment
+
+####Ruby Setup
+you'll need Ruby installed, and either rvm or rbenv to manage your Ruby versions.
+If you don't, follow the directions below:
+
+Install rbenv, which is a tool that helps install/manage versions of Ruby (Note: make sure and follow the brew's post-install instructions):
+
+```
+$ brew install rbenv
+```
+
+And follow the initialization instructions for rbenv, provided by brew
+
+```
+$ rbenv init
+```
+
+Using rbenv install ruby:
+
+```
+$ rbenv install 2.3.0
+```
+
+####Service and Facility Locator Setup
+Clone the repository:
+
+```
+git clone https://github.com/department-of-veterans-affairs/service-and-facility-locator.git
+```
+
+Change directory to service-and-facility-locator:
+
+```
+cd service-and-facility-locator
+```
+
+Install gems and dependencies:
+
+``` 
+bundle install
+```
+
+Create database and load schema:
+
+```
+bundle exec rake db:create 
```

Comment:
Getting the following errors on first install, and if I manually drop the database.

```bash
(~/GitHub/service-and-facility-locator)
>bundle exec rake db:create
services_and_facilities already exists
services_and_facilities already exists

(~/GitHub/service-and-facility-locator)
>bundle exec rake db:drop

(~/GitHub/service-and-facility-locator)
>bundle exec rake db:create
services_and_facilities already exists
```